src/main/java/com/durov/maks/cinema/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,29 @@
+package com.durov.maks.cinema.security;
+
+import com.durov.maks.cinema.model.User;
+import com.durov.maks.cinema.service.UserService;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+    private final UserService userService;
+
+    public CustomUserDetailsService(UserService userService) {
+        this.userService = userService;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
+        User user = userService.findByEmail(email);
+        org.springframework.security.core.userdetails.User.UserBuilder userBuilder;
```

Comment:
```suggestion
        UserBuilder userBuilder;
```
давай винесемо це в імпорт для покращення кодстайлу

Reply:
зрозумів